### ISSUE TITLE
labels for the test set

### ISSUE THREAD

**Issue by zhiyunfan** [joonson/voxconverse#7]
The labels for the test set will be released to public after the VoxCeleb Speaker Recognition Challenge in October 2020. How can we download the labels for the test set? 
Looking forward to your reply.

**Comment by ThGkasios** [comment]
Looking into this as well, the transcripts are in the /dev/ github file. I'll write a script to create the JSON to do training with, as you probably have to do it yourself.

**Comment by scalfs** [comment]
I'm interested in the labels for the test set as well

**Comment by JaesungHuh** [comment]
The test set labels are now released.

**Comment by hbredin** [comment]
Thanks @JaesungHuh for sharing the test set labels.

For comparison with official results reported [here](https://www.robots.ox.ac.uk/~vgg/data/voxceleb/interspeech2020.html), can you please confirm that these were computed on the subset of 232 files for which labels are available and not on the whole set of 312 files shared initially?  This is important for the speaker diarization community to make sure we are not comparing apples and oranges.

cc @fnlandini @desh2608

**Comment by JaesungHuh** [comment]
@hbredin Thanks for the question. Yes, the released Voxconverse test set are subset of 232 files from the whole set of 312 files shared initially. We did another few rounds of check to make labels more accurate and removed some files which annotators couldn't be 100% sure of their annotation. Please use this version from now on.

**Comment by hbredin** [comment]
Thanks for clarifying.

What should we call this version in publications: VoxConverse 2021 ? VoxConverse v0.0.2?

**Comment by JaesungHuh** [comment]
I'll re-open this issue for other people to see. I have to discuss co-authors about this, but I think either is fine. Will let you know if the term fixed.

**Comment by JaesungHuh** [comment]
We've recently released ver 0.3, fixing some of the errors in the test set labels. Please call "VoxConverse 0.3" when you use this dataset.

**Comment by hbredin** [comment]
Thanks for the heads-up @JaesungHuh.

Switching reference labels from 0.2 to 0.3 did ["improve"](https://huggingface.co/pyannote/speaker-diarization/commit/e313d0fafef636b012065d5e7711227d327908a5) my baseline by a whooping 2.8% (relative) in terms of speaker confusion rate. That is not negligible.

**Comment by JaesungHuh** [comment]
Yes. We found these errors during the preparation for this year's [VoxSRC workshop](http://mm.kaist.ac.kr/datasets/voxceleb/voxsrc). I'll re-open this issue to let everyone know about this. I apologize for any inconvenience.

**Comment by ahmadikalkhorani** [comment]
Where can I find the link to the video files?

**Comment by folalafish** [comment]
Where should I download the video file that corresponds to the audio file?

**Comment by shamilajeewantha** [comment]
> Where should I download the video file that corresponds to the audio file?

The links are there in the readme
https://www.robots.ox.ac.uk/~vgg/data/voxconverse/data/voxconverse_test_wav.zip
https://www.robots.ox.ac.uk/~vgg/data/voxconverse/data/voxconverse_dev_wav.zip

**Comment by Euai2002** [comment]
How to get the Video face id in data
